scripts/pi-hole/php/loginpage.php

```diff
@@ -10,7 +10,11 @@
   <div class="panel panel-default">
     <div class="panel-heading">
       <div class="text-center">
-        <img src="img/logo.svg" alt="" style="width: <?php if ($boxedlayout) { ?>50%<?php } else { ?>30%<?php } ?>;">
+        <img src="img/logo.svg" alt="" style="width: <?php if ($boxedlayout) {
+            ?>50%<?php
```

Comment:
IMHO these kind of changes shouldn't be made. It's quite handy to use one-liners here

Reply:
This was made by PHP CodeSniffer, I have not manually modified a single line in this PR.

Reply:
Yeah, we need to tweak the config for sure. The defaults don't cover everything.

I can't find a project I was using to make the config generation easy... I'll try to find it because the defaults clearly cannot cover every case.

Reply:
Mind that any non-standard configuration will make the entire thing harder. Right now, it is just an still fairly easily to memorize call `phpcs --standard=PSR12 ...`

Reply:
Well, the idea is that we don't remember anything. Instead we have scripts that we run. For the JS part this is easily done, but we could also add the phpcs script in the package.json file later.

Reply:
As long as something will be fairly easily doable in the end, this will be fine with me. I'd prefer to be still able to develop from machines not having a web browser and/or php installed locally, but nothing is forbidden from the beginning here, everything is open for discussion.

Reply:
We can just have it to run on CI, that's OK too. As long as the patches are kept squashed, this is viable too.